packages/app-project/src/shared/components/ProjectStatistics/ProjectStatistics.js

```diff
@@ -61,18 +61,22 @@ function ProjectStatistics (props) {
           />
           <NumbersGrid>
             <Stat
+              className="test-stat-project-statistics-volunteers"
```

Comment:
Is there a reason this is `className` rather than `data-testid` for use with [getByTestId](https://testing-library.com/docs/queries/bytestid/)? I'm just curious because these classnames aren't used for anything other than to select one specific element in the dom.

Reply:
Yeah - the Stats component can't accept a `data-testid` unless I modify that component itself to accept it as a parameter.. I think because it's only for testing purposes injecting an identifier through the `className` attribute serves the purpose without touching another component.